kolibri/content/api.py

```diff
@@ -24,6 +26,28 @@ class ChannelMetadataCacheViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.ChannelMetadataCache.objects.all()
 
+    def destroy(self, request, pk=None):
```

Comment:
I added a new permission class `OnlyDeviceOwnerCanDelete` to the Content module. It allows any user to perform 'safe methods', but only superusers can delete. Is this the right approach?